server/lib/rippled.js

```diff
@@ -37,16 +42,24 @@ const formatPaychannel = d => ({
   settleDelay: d.SettleDelay,
 });
 
-// generic RPC query
-const query = options => {
+const executeQuery = (url, options) => {
   const params = { ...options, headers: { 'X-User': HOSTNAME } };
-  return axios.post(URL, params).catch(error => {
+  return axios.post(url, params).catch(error => {
     const message = error.response && error.response.data ? error.response.data : error.toString();
     const code = error.response && error.response.status ? error.response.status : 500;
-    throw new utils.Error(`URL: ${URL} - ${message}`, code);
+    throw new utils.Error(`URL: ${url} - ${message}`, code);
   });
 };
 
+// generic RPC query
+function query(...options) {
+  return executeQuery(URL, ...options);
+}
+
+function adminQuery(...options) {
```

Comment:
I would call this `queryP2P` and `queryFH`.

Reply:
What does FH stand for?

Reply:
full history

Reply:
perhaps `queryReporting` and `queryP2P`? Or even `query` and `queryP2P`?

Reply:
The Full History is true in this specific case, but not true in general (If the config changes, that could easily become false)

Reply:
I think `query` and `queryP2P` makes the most sense to me